canonical-web-and-design/ubuntu.com
# Add support and esm expiry dates to release table

## Done
Added `support` and `esm` expiry dates to the release table

## QA

- dotrun clean
- dotrun exec alembic upgrade head
- all migrations should apply correctly
